docs/examples/micropayment.rst

```diff
@@ -17,14 +17,14 @@ Alice is the sender and Bob is the recipient.
 Alice only needs to send cryptographically signed messages off-chain
 (e.g. via email) to Bob and it is similar to writing checks.
 
-Alice and Bob use signatures to authorise transactions, which is possible with smart contracts on Ethereum.
+Alice and Bob use signatures to authorize transactions, which is possible with smart contracts on Ethereum.
```

Comment:
At some point the documentation was only using British spelling. Are you sure these are the minority? Otherwise we are just making it less consistent.

Reply:
Good point. I approved because in the past we agreed to switch to -ize since it's acceptable in British too, but yeah, better not just change a few instances randomly.

Fortunately seems it's not a problem here. These are the only two instances of -ise/-ize in these two files.